scripts/docker-cross-tag.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2024-2024  The DOSBox Staging Team
+
+usage() {
+    printf "%s\n" "\
+Usage: $0 FUNC OPTIONS
+
+Utility script to assist with docker image tags
+
+FUNC must be one of:
+  tag-exists         Test if image tag exists in GHCR.
+                     Takes two options: IMAGE_NAME, TAG.
+
+  hash-files         Compute a hash of specified files for use in a docker image tag
+                     docker image tag. Command takes one or more file paths as options.
+"
+}
+
+tag_exists_in_ghcr() {
+    local image="$1"
+    local tag="$2"
+
+    # jq and curl are required
```

Comment:
Maybe move this comment to the top into a "prerequisites" section?

Reply:
Done. I've moved all prerequisite checking to a function as well.